src/App.js

```diff
@@ -1,26 +1,42 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [moviesList, refreshMoviesList] = useState(data);
```

Comment:
```suggestion
  const [moviesList, setMoviesList] = useState(data);
```
this is a common practice to use `set` prefix for state changing functions